### DISCUSSION TITLE
Ability to hide CW metrics function in UI

### DISCUSSION THREAD

**Discussion by ainapudi** [grafana/grafana#44942] [Data Sources]
We do not expose CW metrics to Grafana users. We want the ability to hide metrics from the dropdown when selecting the datasource in the UI.

**Comment by ainapudi** [comment]
If we cannot remove/hide the CW Metrics, can CW Logs be the selected by default instead of CW Metrics?

**Comment by sarahzinger** [comment]
Hi @ainapudi just curious is the ask here to only enable Cloudwatch Logs and not Cloudwatch Metrics?

**Comment by sarahzinger** [comment]
Hi @ainapudi just wanted to give you an update that as part of work to move from [discussions to issues](https://github.com/grafana/grafana/discussions/73424), I've opened up an [issue](https://github.com/grafana/grafana/issues/77950) from this discussion for us to consider for future work. 

Thank you for the suggestion!
